config/examples/Creality/CR-10 V3/Configuration.h

```diff
@@ -1187,17 +1187,17 @@
  *     |    [-]    |
  *     O-- FRONT --+
  */
-#define NOZZLE_TO_PROBE_OFFSET { 47, 0, 0 }
+#define NOZZLE_TO_PROBE_OFFSET { 46, 2, 0 }
 
 // Most probes should stay away from the edges of the bed, but
 // with NOZZLE_AS_PROBE this can be negative for a wider probing area.
-#define PROBING_MARGIN 10
+#define PROBING_MARGIN 45
 
 // X and Y axis travel speed (mm/min) between probes
-#define XY_PROBE_FEEDRATE (150*60)
+#define XY_PROBE_FEEDRATE (200*60)
 
 // Feedrate (mm/min) for the first approach when double-probing (MULTIPLE_PROBING == 2)
-#define Z_PROBE_FEEDRATE_FAST (6*60)
+#define Z_PROBE_FEEDRATE_FAST (14*60)
```

Comment:
Max values for smooth operation and good BLtouch accuracy

Reply:
in some cases additional speed will reduce accuracy beyond the acceptable 50 micron deviation.

Reply:
It has been test with multiple attempts at same speeds, multiple speeds, and always doing M48 probe test.
With those speeds, the max deviation from 5 attempts was about 0,002498

Reply:
just because something works for you does not mean it will be the same for someone else there are many variants of the bltouch and many other types of probes.

Reply:
In was test I was going up to 22*60 with still 0,0036 max dev on 5 attemps. So I think it's a fair margin.